### DISCUSSION TITLE
Welcome to anynode Discussions!

### DISCUSSION THREAD

**Discussion by lks-ai** [lks-ai/anynode#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by PrometheusDante** [comment]
Hello there,

First of all, thank you very much for this ingenious node!
<details>
<summary><b>Intro</b></summary>
<i>About me... I'm mainly building workflows trying to make ComfyUI more accessible to beginners and to illustrate the power of structure and UI design even in more complex projects, which is potentially interesting to more advanced users as well. I think most workflows uploaded on civitai are a bit unstructured and chaotic, too lacking in descriptions and explanations throughout, so that everyone downloading them has to untangle the entire spaghetti knot of "user friendly fire" first, only to be left with a very biased/over-personalized approach to a simple problem and many questions. 
My workflows aim to be as general, modular and simple as I can make them with a clear purpose and as much explanation as I can provide to answer most potential questions in advance. Basically trying to provide a lake, a fishing rod and a guide on "How fish?" in one :D<br>
 <br>
At the moment I'm working on a series of workflows to facilitate the entire process of custom character card creation for <b>SillyTavern</b>, which I'll be able to release some time next week I think.</i>
</details>

**Questions:**

- Is there a place where users can share their successful node creations with the prompt used and LLM consulted?
  So far I didn't try much, but wasn't managing to get anything working other than the tutorial "multiply by 5" example :D

**Ideas:**

- To increase user engagement, help the **_anynode_** project grow and improve, as well as expand, having a way for users to share their created nodes' codes with you, so that you can include them in the project or an extra branch potentially would be interesting. Maybe having them in an extra branch for testing and feedback collection to then include the most useful ones in the main branch, might also help users without access to very powerful coding LLMs or users that are too unfamiliar with prompting for nodes.
- A list of recommended LLMs could prove useful as well. I got `codellama:7b` for example, which is very small and seemed good for giving anynode a try.

**Comment by prabinpebam** [comment]
Hi,
Thanks you so much for this node.
I'm struggling to really push the capabilities of the anyNode as it's very hard to control the prompt and iterate on it.
I believe if you break the prompt node and the python node into 2 separate nodes, there will be such great control on what I can run.

As for prompt node, there are some great ones already out there like Ollama custom nodes. I just need a node where I can paste python code directly and it just runs that code. That way I can write my own code or generate the code from another node.

Thanks so much.

**Reply by prabinpebam** [reply]
In case anyone stumbles into this chat. I made the node which I wanted. Anode where you can paste a custom python code directly and execute it. It's very limited in capability as it currently takes only string input and gives string output. I wish someone more capable make it a true anyPython node. Here's the repository [anyPython](https://github.com/prabinpebam/anyPython)
